pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -309,10 +309,12 @@ def set_registration_controls_sensitive(self, sensitive, include_register_button
     # authentication related signals
 
     def on_account_radio_button_toggled(self, radio):
```

Comment:
In the commit message: ``authetication`` -> ``authentication``.

Reply:
Fixed, thanks! :)